package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "edx-ora2",
-  "version": "0.0.1",
+  "version": "0.2.0",
```

Comment:
I described this in the PR comment. It seemed broken that this had never been bumped. However, this came from the feature branch, so I could move it back as that's where the new functionality is coming.